Controllers/db/index.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require('mysql');
+const pool = mysql.createPool({
+  connectionLimit: 10,
+  host: process.env.DB_HOST,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_NAME,
+  port: process.env.DB_PORT,
+  multipleStatements: true,
```

Comment:
Do we really require multipleStatements in this project??

Reply:
OK ill remove that